### PR TITLE
Adds embeddings migration and cleanup mutations temporarily

### PR DESCRIPTION
**Pull request by valeksiev**
### Describe the background of your pull request

Implements https://github.com/alkem-io/virtual-contributor-ingest-space/issues/30

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/domain/communication/room/room.service.events.ts [hunk 1]
```diff
@@ -117,7 +117,7 @@ export class RoomServiceEvents {
         const result = await this.virtualContributorService.askQuestion(
           chatData,
           agentInfo,
-          spaceNameID
+          spaceID
```
Comment (valeksiev) [inline src/domain/communication/room/room.service.events.ts:120]: context space should be loaded using UUID as well - this was missed in 4010

Comment (techsmyth) [reply src/domain/communication/room/room.service.events.ts:120]: Good catch

File: src/domain/community/virtual-contributor/virtual.contributor.resolver.mutations.ts [hunk 2]
```diff
@@ -1,24 +1,173 @@
...
-import { AuthorizationPrivilege } from '@common/enums';
+import { AuthorizationPrivilege, ConfigurationTypes } from '@common/enums';
 import { AgentInfo } from '@core/authentication.agent.info/agent.info';
 import { AuthorizationService } from '@core/authorization/authorization.service';
 import { IVirtualContributor } from './virtual.contributor.interface';
 import {
   DeleteVirtualContributorInput,
   UpdateVirtualContributorInput,
 } from './dto';
+import { ChromaClient } from 'chromadb';
```
Comment (techsmyth) [inline src/domain/community/virtual-contributor/virtual.contributor.resolver.mutations.ts]: Please move these two mutations down to be on the AI Server, not on VIrtual Contributor - even if they are going to be shortly removed again.

**Review by techsmyth — DISMISSED** [review]
Good catch re also context UUID. The two mutations to migrate need to be on AI Server module, not in the domain - even if they will be shortly removed again. 

Seeing ChromaDb dependencies coming in for me is a clear sign we need to get this out into a separate microservice!

**Pull request merged**